tracee-ebpf/main.go

```diff
@@ -233,12 +233,14 @@ Possible options:
 [artifact:]all                     capture all of the above artifacts.
 
 dir:/path/to/dir        path where tracee will save produced artifacts. the artifact will be saved into an 'out' subdirectory. (default: /tmp/tracee).
+profile                 creates a runtime profile of captured artifacts and their metadata for forensics use. the profile will be saved into the 'out' directory. (default: /tmp/tracee/out/tracee.profile).
```

Comment:
needless to say where it's captured to since it's documented for all the capture flags. also none of the other ones say it

Reply:
OK

Reply:
https://github.com/aquasecurity/tracee/pull/725/commits/718b24868f23b657de987413eebf28c956137c67

Reply:
not just the default also the mention of the output directory. like the other descriptions